integration/flink-native-listener/gradle.properties

```diff
@@ -1,5 +1,6 @@
 jdk8.build=true
 version=1.27.0-SNAPSHOT
 # Set the version of Flink to use - currently snapshot
-flink.version=2.0-preview1
+#flink.version=2.0-preview1
+flink.version=2.0-SNAPSHOT
```

Comment:
works on snapshot as it requires flink core notifications with PlannerQueryOperation events.